.github/workflows/pull-request-schedule.yaml

```diff
@@ -0,0 +1,103 @@
+name: Pull Request Scheduler
+on:
+  schedule:
+  - cron: '*/10 * * * *'
+  workflow_dispatch: {}
+
+env:
+  BACKEND: gke
+  CLUSTERS_PER_PR: "2"
+  MAX_CLUSTERS: "5"
+  OWNER: ${{ github.repository_owner }}
+  LABEL_QUEUE: pr-test-queue
+  LABEL_TRIGGER: pr-test-trigger
+
+jobs:
+  determine-capacity:
+    name: Determine Capacity
+    runs-on: ubuntu-latest
+    outputs:
+      # The number of pull requests we can start testing now.
+      capacity: ${{ steps.calculate-capacity.outputs.capacity }}
+    env:
+      GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
+    steps:
+    - name: Checkout KubeCF
+      # This is only needed to find the GitHub script in the find-jobs step.
+      uses: actions/checkout@v2
+    - name: Checkout catapult
+      uses: actions/checkout@v2
+      with:
+        repository: SUSE/catapult
+        path: catapult
+
+    # set GKE secret. TBD: use vault instead
+    - name: Setup Cloud Provider Credentials
+      run: |
+        json_file="$(mktemp)"
+        cat > "${json_file}" <<< "${GKE_CRED_JSON}"
+        echo "GKE_CRED_JSON=${json_file}" >> "${GITHUB_ENV}"
```

Comment:
Does this respect newline characters correctly?

Reply:
There are no new line characters involved: `GKE_CRED_JSON` (as exported to `GITHUB_ENV`) is the path to the temporary file we created.  (There's a new line created by `echo` but I don't think that's not what you meant.)

Reply:
doh true, it's a file. That answers my question.